# HG changeset patch
# User Jacob Dawid <jacob.dawid@gmail.com>
# Date 1344070438 -7200
#      Sat Aug 04 10:53:58 2012 +0200
# Branch gui
# Node ID 093961d9ebed837408c8f0179b6562d68866ee63
# Parent  d367f5ab0838ade121176fca065ae8118d07e5a1
Fixed self-assignment bug found by Torsten.

* find-dialog.cc: Fixed self-assignment in constructor.

diff --git a/gui/src/m-editor/find-dialog.cc b/gui/src/m-editor/find-dialog.cc
--- a/gui/src/m-editor/find-dialog.cc
+++ b/gui/src/m-editor/find-dialog.cc
@@ -66,17 +66,17 @@ find_dialog::find_dialog (QsciScintilla*
   _button_box->addButton (_more_button, QDialogButtonBox::ActionRole);
 
   _extension = new QWidget (this);
   _whole_words_check_box = new QCheckBox (tr ("&Whole words"));
   _regex_check_box = new QCheckBox (tr ("Regular E&xpressions"));
   _backward_check_box = new QCheckBox (tr ("Search &backward"));
   _search_selection_check_box = new QCheckBox (tr ("Search se&lection"));
 
-  edit_area = edit_area;
+  _edit_area = edit_area;
   connect (_find_next_button, SIGNAL (clicked ()), this, SLOT (search_next ()));
   connect (_more_button, SIGNAL (toggled (bool)), _extension, SLOT (setVisible (bool)));
 
   QVBoxLayout *extension_layout = new QVBoxLayout ();
   extension_layout->setMargin (0);
   extension_layout->addWidget (_whole_words_check_box);
   extension_layout->addWidget (_regex_check_box);
   extension_layout->addWidget (_backward_check_box);
